### DISCUSSION TITLE
Where is Qt5 in 4.4.3.10 release?

### DISCUSSION THREAD

**Discussion by gjf** [c0re100/qBittorrent-Enhanced-Edition#383] [General]
Taking into account "Qt6 Known issue: The text on the progress bar may have wrong color or not appear at all" Qt5 version was the best as for 4.4.2.10.

Why 4.4.3.10 does not include it?

**Comment by gjf** [comment]
Also please note 4.4.2.10 "no-Qt" version lags and freezes on Win11x64. So Qt5 was really the only option without any problem.

**Reply by c0re100** [reply]
> Also please note 4.4.2.10 "no-Qt" version lags and freezes on Win11x64. So Qt5 was really the only option without any problem.

"no-Qt" version=Qt5
